Sources/Kitura/CodableRouter.swift

```diff
@@ -462,6 +564,26 @@ extension Router {
         }
     }
 
+    // DELETE w/Optional Query Parameters
+    fileprivate func deleteSafely<Q: QueryParams>(_ route: String, handler: @escaping (Q?, @escaping ResultClosure) -> Void) {
+        delete(route) { request, response, next in
```

Comment:
Now that #1281 is merged, this needs a call to `registerDeleteRoute()` at the start of the function.